src/lib/gossip_net/fake.ml

```diff
@@ -80,7 +80,8 @@ module Make (Rpc_intf : Coda_base.Rpc_intf.Rpc_interface_intf) :
                     Incoming.wrap ~data:msg
                       ~sender:(Sender.Remote (sender.host, sender.peer_id)))
                 in
-                Strict_pipe.Writer.write intf.broadcast_message_writer msg ) )
+                Strict_pipe.Writer.write intf.broadcast_message_writer
+                  (msg, Fn.const ()) ) )
```

Comment:
Do we not need to reorganize the fake network rebroadcast logic to use the new `valid_cb` model instead of the explicit broadcast model?

Reply:
As long as the rebroadcast logic works, I think it's still the right thing to do in the fake network.